SalesforceSDK/Core/Auth/OAuth2.cs

```diff
@@ -323,30 +298,53 @@ public static string ComputeFrontDoorUrl(string instanceUrl, string displayType,
         /// <returns>Boolean based on if the refresh auth token succeeded or not</returns>
         public static async Task<Account> RefreshAuthToken(Account account)
```

Comment:
I am thinking lets make this call now and change it. Since we are releasing our first version for Windows 10, it seems like good time to do this change.

@bhariharan @smcnulty-sfdc What are your thoughts on changing method names given the fact that we will be releasing our first version of the Widnwos SDK for Windows 10?